### DISCUSSION TITLE
Welcome to x-ui-pro Discussions!

### DISCUSSION THREAD

**Discussion by GFW4Fun** [GFW4Fun/x-ui-pro#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Farhooralz** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.


hi man first i wanna thank you for your hard work and also i was wondering, since i'm using marzban can i configure the nginx and the haproxy for my server that it will not be banned or filtered ? and i want to know that can you do all of this on marzban panel as well i mean all the things you did in this x-ui pro ?
